measure_extinction/utils/merge_iue_spec.py

```diff
@@ -30,7 +27,7 @@
     args = parser.parse_args()
 
     # incomplete - code to read in the files needed
-    # use_small capabiliy needed
+    # use_small capability needed
     # recalibration by Massa & Fitzpatrick IDL code needs to be recoded in python
```

Comment:
Seems like you should open another issue for this, if not already done.

Reply:
Already there for sometime.  #4.